packages/editor/src/client/blocks/richTextBlock/toolbar/linkMenu.tsx

```diff
@@ -36,12 +36,17 @@ export function LinkMenu() {
   const [selection, setSelection] = useState<Range | null>(null)
 
   const [isValidURL, setIsValidURL] = useState(false)
+  const [isValidMail, setIsValidMail] = useState(false)
   const isDisabled = !url || !title
 
   const {t} = useTranslation()
 
   useEffect(() => {
-    validateURL(url).then(value => setIsValidURL(value))
+    if (prefix === prefixType.mailto) {
+      validateMail(url).then(value => setIsValidMail(value))
```

Comment:
Guys, let's rather implement `validateLink(prefix, url)` and only use `isValidLink`.